python/lsst/ts/MTAlignment/mockT2sa.py

```diff
@@ -32,6 +38,11 @@ def __init__(self, ip="172.17.0.2"):
         await asyncio.wait_for(server.wait_closed(), timeout=5)
 
     async def response_loop(self, reader, writer):
+        """
+        Listens for messages from the CSC and responds either with canned text or
+        by invoking a method from self.response_dict.
+        """
+
         print("Response Loop begins")
         while True:
```

Comment:
You may want to add a `run_loop_flag` that you can switch off and stop the loop instead. This will make it easier to shut the server down gracefully.